Adding error code descriptions and examples to Budget API endpoints & other minor changes

barodapride opened the pull request.

barodapride committed:
Update getting-started.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-adjustments.md

charlieconcur commented on src/api-reference/budget/budget-header.md:
````diff
@@ -581,4 +581,34 @@ Name | Type | Format | Description
+```
+```json
+{
+  "status" : false, 
````
Something is wrong with either the markup or the JSON here.

barodapride replied:
This should be fixed now, there was a missing " way above in the document

charlieconcur replied:
Excellent.

charlieconcur commented:
We need to fix the JSON issues present and have a peer reviewer approve the changes.

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

charlieconcur commented on src/api-reference/budget/getting-started.md:
```diff
@@ -66,7 +66,7 @@ Examples:
 HTTPie:
```
There should be no links for specific tooling. Please follow the example in the [Documentation Template](https://github.concur.com/developer/api/blob/master/documentation-template.md).

charlieconcur commented:
@barodapride are you a SAP Concur employee or customer?

barodapride commented:
@charlieconcur yes I work on the budget team. Somebody should review this pull request soon.

jtbigwoo commented:
- [x] Proofread documentation changes for spelling and grammatical errors
- [x] Used Markdown code blocks for all applicable examples using code
- [x] Used relative links when linking to other documentation on Developer Center
[N/A] If creating new API reference pages, followed the guidelines stated in the wiki
[N/A] If moving pages, added page redirects so users are redirected to the new page

These changes are approved, except for the change requested by Charlie.

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-category.md

barodapride committed:
Update budget-adjustments.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update budget-header.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update fiscal-year.md

barodapride committed:
Update getting-started.md

barodapride commented:
Updated documentation to get rid of HTTPie examples - replaced with http markdown. Also moved the request parameter info to the top as per the example documentation.

charlieconcur commented:
I'm going to be a stickler here: We need all documentation to adhere to the documentation template which you can find at https://github.com/concur/developer.concur.com/blob/receipt-updates-and-polish/src/document-template.markdown. I am happy to do a co-working session if that will help you bring things into alignment.

barodapride commented:
@mmeenapriya

barodapride commented:
@charlieconcur I need some clarification as to what is not up to par. I more or less followed the template but there are some parts that are a bit different. Is there some model documentation somewhere I can see for an example?

charlieconcur commented:
The documentation needs to follow the template flow. We are in the process of updating all documentation to be consistent.

barodapride commented:
Closing, changes were moved into another request
